### DISCUSSION TITLE
Port to Eto

### DISCUSSION THREAD

**Discussion by rafntor** [Mapsui/Mapsui#1524] [Show and tell]
Hi, and thanks for an impressive library !

I have made a port targeting [Eto.Forms](https://github.com/picoe/Eto) ;
Would you welcome a PR and bring this into your project ? (I would try to help out and assist in issues within reason)

- MacOS
![mac-demo](https://user-images.githubusercontent.com/44544090/155893193-3b7363ce-a72b-4dd8-8267-14ca26614f9e.gif)

- Gtk (on Fedora)
![gtk-demo](https://user-images.githubusercontent.com/44544090/155893198-959f6cc4-a9b7-4c4c-8caa-a81efa0e1109.gif)

- Wpf
![wpf-demo](https://user-images.githubusercontent.com/44544090/155893201-e9a9658a-3105-4860-b60d-3b06e9ec0613.gif)

**Comment by inforithmics** [comment]
I would certainly accept a Pull Request for Eto.Forms. 

I think the Target would be Mapsui.UI.Eto
I can help you to configure it for the gitub actions and appveyor build. But you have to make first an intial pull request.

**Reply by rafntor** [reply]
Great, thanks ! Will do ;-)
